### DISCUSSION TITLE
Welcome to PowerShell Practice And Style Discussions!

### DISCUSSION THREAD

**Discussion by Jaykul** [PoshCode/PowerShellPracticeAndStyle#165] [Announcements]
## 👋 Welcome!
  We’re trying Discussions as an alternative to Issues for some of those questions where a solid consensus hasn't made it into the book yet. We hope that you will:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we build together 💪.

**Comment by hrxn** [comment]
Hey everyone! 😄
